### DISCUSSION TITLE
Add "Copy/Duplicate" Button to Builder

### DISCUSSION THREAD

**Discussion by hany-cmd** [filamentphp/filament#14225] [Ideas]
It would be helpful to have a "Copy" or "Duplicate" button next to the "Delete" button in the form builder. This would allow users to quickly duplicate form components, saving time when working with similar fields. Currently, recreating components manually is time-consuming.

![qweqw23123](https://github.com/user-attachments/assets/f2678cc8-c449-48da-b89f-9c1d6c21dd2f)

**Comment by willcastillo** [comment]
Hey... What would be the difference between what you are asking and using `->cloneable()`?

**Comment by hany-cmd** [comment]
Hi, thanks for your input! I need the ‘Copy’ button to duplicate a block and paste it onto another page. This would streamline the process and allow for quicker replication of components across different forms or sections.
